arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/StackFragment.java

```diff
@@ -263,17 +269,17 @@ public Trace trace(Trace trace) {
         .pStackJumpDestinationVettingRequired(
             this.jumpDestinationVettingRequired) // TODO: confirm this
         // Exception flag
-        .pStackOpcx(Exceptions.invalidCodePrefix(exceptions))
-        .pStackSux(Exceptions.stackUnderflow(exceptions))
-        .pStackSox(Exceptions.stackOverflow(exceptions))
-        .pStackMxpx(Exceptions.memoryExpansionException(exceptions))
-        .pStackOogx(Exceptions.outOfGasException(exceptions))
-        .pStackRdcx(Exceptions.returnDataCopyFault(exceptions))
-        .pStackJumpx(Exceptions.jumpFault(exceptions))
-        .pStackStaticx(Exceptions.staticFault(exceptions))
-        .pStackSstorex(Exceptions.outOfSStore(exceptions))
-        .pStackIcpx(contextExceptions.invalidCodePrefix())
-        .pStackMaxcsx(contextExceptions.codeSizeOverflow())
+        .pStackOpcx(tracedException == Exceptions.INVALID_OPCODE)
+        .pStackSux(tracedException == Exceptions.STACK_UNDERFLOW)
+        .pStackSox(tracedException == Exceptions.STACK_OVERFLOW)
+        .pStackMxpx(tracedException == Exceptions.MEMORY_EXPANSION_EXCEPTION)
+        .pStackOogx(tracedException == Exceptions.OUT_OF_GAS_EXCEPTION)
+        .pStackRdcx(tracedException == Exceptions.RETURN_DATA_COPY_FAULT)
+        .pStackJumpx(tracedException == Exceptions.JUMP_FAULT)
+        .pStackStaticx(tracedException == Exceptions.STATIC_FAULT)
+        .pStackSstorex(tracedException == Exceptions.OUT_OF_SSTORE)
+        .pStackIcpx(tracedException == Exceptions.INVALID_CODE_PREFIX)
+        .pStackMaxcsx(tracedException == Exceptions.CODE_SIZE_OVERFLOW)
```

Comment:
Here you can do a static import of `Exceptions` to get rid of all the `Exceptions.` prefixes

Reply:
You should also add somwhere where it makes sense, e.g. in the tracing here, a check that `exceptions` and `tracedExceptions` satisfy the properties from the issue.